src/client/components/measures-tile/measures-tile.tsx

```diff
@@ -74,23 +96,36 @@ export class MeasuresTile extends React.Component<MeasuresTileProps, MeasuresTil
 
     if (!essence.isFixedMeasureMode()) {
       icons.push({
-        name: 'multi',
-        onClick: clicker.toggleMultiMeasureMode,
-        svg: require('../../icons/full-multi.svg'),
-        active: multiMeasureMode
-      });
+          name: 'multi',
+          onClick: clicker.toggleMultiMeasureMode,
+          svg: require('../../icons/full-multi.svg'),
+          active: multiMeasureMode
+        },
+        {
+          name: 'search',
+          ref: 'search',
+          onClick: this.toggleSearch.bind(this),
+          svg: require('../../icons/full-search.svg'),
+          active: showSearch
+        }
+      );
```

Comment:
This is wrong. Only the multi mode icon should be in this if statement. Right now Measure search does not show up on totals